python/server/qgsrequesthandler.sip

```diff
@@ -61,57 +61,52 @@ class QgsRequestHandler
     // virtual void setGetFeatureResponse( QByteArray* ba ) = 0;
 
     //! @note not available in Python bindings
```

Comment:
@nyalldawson I don't know, this has been left unchanged. My felling it is that they are ad-hoc methods used only in wfs for streaming getfeature request. IMHO they brings very little functionality as they just encapsulate write/flush to response and I think also that they are misplaced.

I have planned a PR to make some cleanup when this one will be merged.